packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -51,20 +51,36 @@ class FramesBarPlotly {
         // Hide ticks by using font color of bgColor.
         tickfont: Font(
           color: 'white',
+          size: 1,
         ),
         rangemode: 'nonnegative',
         autorange: true,
       ),
       yaxis: AxisLayout(
         title: 'Milliseconds',
-        fixedrange: true,
+        tickformat: '.0f',
```

Comment:
it would be nice to add a top level boolean flag so it is easy for someone else to turn off the log scale if they want.
e.g. add

bool useLogScale = true;
and only add these options if that is the case. That way it will be easy for someone to give Tao a custom build with these sorts of features on or off in the future.

Reply:
Done.